orttraining/orttraining/training_ops/cpu/aten_ops/aten_op_config.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <vector>
+#include <string>
+#include <unordered_map>
+
+namespace onnxruntime {
+namespace contrib {
+namespace aten_ops {
+
+// TODO: We need to ATen operator config to pass arguments to PyTorch as well as building gradient graph.
+// Currently these configs are C++ codes below, ideally we can use string/text configs
+// just as derivatives.yaml in PyTorch, and parse that text to generate below configs.
+
+enum ForwardTensorOutputTypeKind {
+  PROPAGATE_FROM_INPUT,
+  CONCRETE_TYPE,
+};
+
+enum BackwardTensorInputKind {
+  GRAD_OUTPUT,
+  FORWARD_INPUT,
+  FORWARD_OUTPUT,
+};
+
+enum ArgumentKind {
+  TENSOR,
+  INT,
+  FLOAT,
+  BOOL,
+  // TODO: need more type, such as list type
+};
+
+// TODO: need to support default attribute value.
+struct ATenOperatorConfig {
+  std::string backward_op_name;
+  std::vector<std::tuple<ArgumentKind, std::string>> forward_argument_configs;
+  std::vector<std::tuple<ArgumentKind, std::string>> backward_argument_configs;
+  std::vector<std::tuple<BackwardTensorInputKind, int>> backward_tensor_input_configs;
+  std::vector<std::tuple<ForwardTensorOutputTypeKind, int>> forward_tensor_output_type_configs;
+  std::vector<int> backward_output_configs;
```

Comment:
not exactly sure what's the usage of backward_output_configs just by looking at the name, and the example below... 
maybe a comment to explain this?